tests/parametric/test_otel_span_with_w3c.py

```diff
@@ -43,20 +43,3 @@ def test_otel_start_span_with_w3c(self, test_agent, test_library):
         assert root_span["resource"] == "operation"
         assert root_span["meta"]["start_attr_key"] == "start_attr_val"
         assert root_span["duration"] == duration_ns
-
-    @irrelevant(context.library == "cpp", reason="library does not implement OpenTelemetry")
-    def test_otel_span_with_w3c_headers(self, test_agent, test_library):
```

Comment:
Could you review the manifest files? For example there is an entry for "Test_Otel_Span_With_W3c" in python.yml

Reply:
Test_Otel_Span_With_W3c is an entry in some manifest files. We can actually remove this whole test class. I can do that now and update the manifest files.